meg_runtime/app.py

```diff
@@ -60,6 +62,26 @@ def get_panel(name):
         """Get a panel by name"""
         return None if not App.__instance else App.__instance.panel(name)
 
+    @staticmethod
+    def refresh_panel(name, **kwargs):
+        """Refresh a panel by name."""
+        instance = App.__instance
+        if instance is None:
+            return None
+        if name in App.PANELS:
+            try:
+                panel_ctor = getattr(ui, name)
+                panel_obj = panel_ctor(**kwargs)
+                instance._panels[panel_obj.get_name()] = panel_obj
+                return panel_obj
+            except Exception as e:
+                Logger.warning(f'MEG UI: {e}')
+                Logger.warning(f'MEG UI: Could not create panel "{name}"')
+        else:
+            Logger.warning(f'MEG UI: Panel does not exist')
+
+        return None
+
     def name(self):
         """Get application name"""
         return App.APP_NAME
```

Comment:
Need version() and also did I name that APP_NAME in App, lol? I think should have just App.NAME and App.VERSION (probably can change the name of App.APP_ICON_PATH to App.ICON_PATH as well). Also should have static methods get_name() and get_version(), that call the instance methods.